numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -1304,7 +1353,7 @@ _pyarray_revert(PyArrayObject *ret)
  * correlate(a2, a1), and conjugate the second argument for complex inputs
  */
 NPY_NO_EXPORT PyObject *
-PyArray_Correlate2(PyObject *op1, PyObject *op2, int mode)
+PyArray_Correlate2(PyObject *op1, PyObject *op2, int mode, npy_intp minlag, npy_intp maxlag, npy_intp lagstep)
```

Comment:
This breaks backwards compatibility for the exported C API. The signature should remain
```PyObject* PyArray_Correlate2(PyObject* op1, PyObject* op2, int mode)```